Makefile

```diff
@@ -28,10 +28,10 @@ compile-cairo: $(CAIRO_TARGETS)
 compile-starknet: $(STARKNET_TARGETS)
 
 cairo_programs/%.json: cairo_programs/%.cairo
-	cairo-compile $< --output $@
+	cairo-compile $< --output $@ || rm $@
```

Comment:
is this ok?

Reply:
Yes, the `|| rm $@` removes the JSON file when `cairo-compile` fails so that running `make compile-starknet` again will retry the compilation (disregarding whether the source file has been modified or not).